src/OpenSage.Mathematics/FloatRange.cs

```diff
@@ -10,5 +12,11 @@ public FloatRange(float low, float high)
             Low = low;
             High = high;
         }
+
+        public float GetValue(Random random)
+        {
+            if (random == null) return High;
```

Comment:
Why exactly do you return **High** here?

Reply:
Well we could also return `Low` here, or any other suggestions?

Reply:
Crash? When is this ever called without a random

Reply:
I'm also curious, does this ever need to be called without a `Random`?

Reply:
Yes, from GameObject->UpdateDrawModuleConditionStates->UpdateConditionState->SetActiveConditionState and there is no proper GameContext with the corresponding Random available. Should I just use a new one instead?

Reply:
But `GameObject` does have a `_gameContext` field available?

Reply:
using a new random now